src/main/service/filterService.ts

```diff
@@ -138,4 +141,16 @@ export class FilterService {
 
     return filterValues;
   }
+
+  public static generateAlphabetObject(): object {
```

Comment:
This can potentially be moved into a helper class and shared with locationService

Reply:
Now moved out into a seperate class